src/list-data/request-action-mapping.ts

```diff
@@ -81,8 +81,8 @@ export const XproMapping: RequestActionMappingI = {
   ],
   CHG: [
     EntityTypes.A,
-    EntityTypes.XP,
-    EntityTypes.XL,
+    EntityTypes.XLP,
+    EntityTypes.XLL,
```

Comment:
1. Should we have both XP and XLP (and XL and XLL)?

2. Could you please fix entity-types.ts? XP should be `CorpTypeCd.XPRO_LIM_PARTNR`

Reply:
1. I was a bit confused about keeping it but it makes sense to keep both codes
2. Will do